printing/genprint.ml

```diff
@@ -118,7 +118,7 @@ struct
   let name = "printer"
   let default wit = match wit with
   | ExtraArg tag ->
-    let name = ArgT.repr tag in
+    let name = try ArgT.repr tag with Assert_failure _ when !Flags.in_debugger -> "UNKNOWN" in
```

Comment:
this should be in the other PR

Reply:
I mixed up the two PRs. Should be fixed.